### ISSUE TITLE
Note: Desustorage is dead, it is now desuarchive.org

### ISSUE THREAD

**Issue by Yoposoc** [zixaphir/appchan-x#984]
Pls change. Thanks.

**Comment by TedofChicago** [comment]
desistoage is dead long live desuarchive

**Comment by Yoposoc** [comment]
Still not fixed. I guess Appchan-x is abandoned.

**Comment by vampiricwulf** [comment]
It pretty much is. I only really go on /f/ and I have no idea what Zixa is up to.

**Comment by TedofChicago** [comment]
dang

**Comment by Yoposoc** [comment]
RIP. Next best alternative? Is it the original chan-x?

**Comment by vampiricwulf** [comment]
Yeah ccd0 is pretty fucking awesome so you can do that, and you can use the oneechan that's being updated to bring back the appchan-x look.
https://www.4chan-x.net/
https://nebukazar.github.io/OneeChan/
If you don't care about mascots: https://nebukazar.github.io/StyleChan/

**Comment by Yoposoc** [comment]
will do. thanks.
